tests/integration/api_container_test.py

```diff
@@ -1255,6 +1255,14 @@ def test_container_cpuset(self):
         inspect_data = self.client.inspect_container(container)
         self.assertEqual(inspect_data['HostConfig']['CpusetCpus'], cpuset_cpus)
 
+    def test_create_with_runtime(self):
```

Comment:
Please add a `@requires_api_version` decorator with the minimum API version for this feature.